src/UCTSearch.cpp

```diff
@@ -58,9 +58,11 @@ constexpr int UCTSearch::UNLIMITED_PLAYOUTS;
 class OutputAnalysisData {
 public:
     OutputAnalysisData(const std::string& move, int visits,
-                       float winrate, float policy_prior, std::string pv)
+                       float winrate, float policy_prior, std::string pv,
+                       float lcb, bool lcb_ratio_exceeded)
     : m_move(move), m_visits(visits), m_winrate(winrate),
-      m_policy_prior(policy_prior), m_pv(pv) {};
+      m_policy_prior(policy_prior), m_pv(pv), m_lcb(lcb),
+      m_lcb_ratio_exceeded(lcb_ratio_exceeded) {};
 
     std::string get_info_string(int order) const {
         auto tmp = "info move " + m_move
```

Comment:
I think you want to add "lcb" here. Without it it's also going to be confusing why the engine plays the "wrong" move.

(After it, it will still be confusing because of the min_visits condition, but that seems unfixable)